dataset/dictionary/items.json5

```diff
@@ -1160,7 +1160,7 @@
     ja: "ヴァソマの実",
     zhCN: "恒素果",
     tags: [ "item", "sumeru" ],
-    pinyins: [{ char: "桓", pron: "huan2" }],
+    pinyins: [{ char: "恒", pron: "heng2" }],
```

Comment:
@SleepyAsh0191 I found another wrong pinyin. Can you check if this is properly fixed?